simde/arm/neon/types.h

```diff
@@ -133,6 +133,19 @@ typedef union {
   #endif
 } simde_float64x1_private;
 
+// [Eric] Add 64bits poly type
```

Comment:
```suggestion
// [Eric] Add 64bits poly type
```
Hey Eric, I appreciate your massive PR! You will receive credit; but I don't think these comments add value to the reader

Reply:
Ok, I'll remove it.